src/plugins/intel_cpu/src/mkldnn_graph_optimizer.cpp

```diff
@@ -1117,17 +1117,29 @@ void MKLDNNGraphOptimizer::FuseConvolutionSumAndConvolutionSumActivation(MKLDNNG
             isSuitableParent2 = isSuitableParent2 && canFuseSum(binConvNode2, graphNode);
         }
 
+        auto checkFusedWith = [](MKLDNNConvolutionNode* conv) -> bool {
```

Comment:
maybe?
```suggestion
        auto checkFusedWithSum = [](MKLDNNConvolutionNode* conv) -> bool {
```

Reply:
[random spot] is it possible to add subgraph test for this case?

Reply:
Agree, done